prow/cluster/plank_deployment.yaml

```diff
@@ -29,13 +29,22 @@ spec:
         role: prow
       containers:
       - name: plank
-        image: gcr.io/k8s-prow/plank:0.17
+        image: gcr.io/k8s-prow/plank:0.18
+        args:
+        - "--dry-run=false"
+        - "--github-bot-name=k8s-ci-robot"
```

Comment:
Agreed. It should either be in the prow config or else a part of the k8s secret, since it's tied to the OAuth token in a sense.